sherpa.spec

```diff
@@ -15,6 +15,18 @@ Requires: openloops
 %endif # isamd64
 %endif # islinux
 
+%if "%{?cms_cxx:set}" != "set"
```

Comment:
It's used in L60-L62
https://github.com/pmillet/cmsdist/blob/53c61350d4db4681e855ee510846d9380c98cda7/sherpa.spec#L60-L62